src/main/java/com/empowerhack/family/memberservice/FamilyMemberServiceApplication.java

```diff
@@ -9,4 +18,61 @@
     public static void main(String[] args) {
         SpringApplication.run(FamilyMemberServiceApplication.class, args);
     }
+
+    @Bean
```

Comment:
Yeah, temporary solution just for playing with while using **In-memory** Database